Packs/MicrosoftExchangeOnline/ReleaseNotes/1_6_1.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### EWS O365
+
+- Modified EWS API module to support the EWSv2 integration. This should have no impact on this integration.
```

Comment:
I think we should change the release notes to some thing more related to the changes in EWSO365 such as. 
Removed deprecated code or metadata and documentation improvments.